docs/api-docs/api-docs.yml

```
@@ -77,6 +84,17 @@ paths:
                         "start": 30
                       }
                     ]
+                Batch response (array of texts):
+                  value:
+                    [
+                      [
+                        { "entity_type": "PERSON", "start": 0, "end": 10, "score": 0.85, "analysis_explanation": null },
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)